inc/saibridge.h

```diff
@@ -85,6 +85,8 @@ typedef enum _sai_bridge_port_type_t
     /** Bridge tunnel port */
     SAI_BRIDGE_PORT_TYPE_TUNNEL,
 
+    /** Bridge port next hop group */
+    SAI_BRIDGE_PORT_TYPE_BRIDGE_PORT_NEXT_HOP_GROUP,
```

Comment:
Can we have this named as SAI_BRIDGE_PORT_TYPE_NEXT_HOP_GROUP ?

Reply:
This is specific to NHG of type bridgeport and hence this name.

Reply:
@JaiOCP Please comment if the extra BRIDGE_PORT in the name can be dropped ?

Reply:
Rational for naming it like is to be explicit in intent that the NHG type is BRIDGE_PORT. Though the attributes are for  bridge port object and it may seem redundant but the NHG are of many type and one used here is of type bridge port.

It would be good to clarify this in comments to avoid such confusion.

Reply:
Updated comments.